### PR TITLE
LPS-198120 Empty error message when disabling object definition translation with translatable field

### PR DESCRIPTION
**Pull request by diogomelo09**
[LPS-198120](https://liferay.atlassian.net/browse/LPS-198120)

### CODE REVIEW

**Comment by diogomelo09** [comment]
ci:test:sf

**Comment by diogomelo09** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
Hi @diogomelo09 !

Does this fix follow the expect behavior? In the expected behavior, we have that the toggle should be inactive to be changed.

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java [hunk 1]
```diff
@@ -1794,6 +1797,20 @@ private ObjectDefinition _publishObjectDefinition(
 
 		objectDefinition = objectDefinitionPersistence.update(objectDefinition);
 
+		for (ObjectField objectField : objectFields) {
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java:1800]: @diogomelo09 Can you use ListUtil.exists here, please? Thx

**Review by gabrielwas — CHANGES_REQUESTED** [review]
@diogomelo09 check the comments, pls! Thx!

**Comment by diogomelo09** [comment]
ci:test:sf

**Comment by diogomelo09** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
@diogomelo09 Probably there's a related failed test on `com.liferay.object.service.test.ObjectFieldLocalServiceTest`. Could you check it? Thx

**Comment by diogomelo09** [comment]
ci:test:sf

**Comment by diogomelo09** [comment]
ci:test:relevant

**Comment by diogomelo09** [comment]
ci:test:relevant

**Comment by evanilson-santana** [comment]
ci:test:object

**Review by gabrielwas — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java [hunk 2]
```diff
@@ -151,7 +150,20 @@ public void testAddCustomObjectField() throws Exception {
 						"a" + RandomTestUtil.randomString()
 					).localized(
 						true
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (diogomelo09) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java]: Done! :heavy_check_mark:

**Comment by diogomelo09** [comment]
ci:test:sf

**Comment by evanilson-santana** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**